tensorflow/compiler/xla/service/gpu/gemm_rewriter.cc

```diff
@@ -567,8 +569,9 @@ class GemmRewriterVisitor : public DfsHloRewriteVisitor {
     }
 
     if (Match(instr,
-              m::AddAnyOrder(GemmOrCublasLtMatmul(&existing_gemm).WithOneUser(),
-                             m::Op(&bias).WithPredicate(is_not_broadcast)))) {
+              m::AddAnyOrder(
+                  GemmOrCublasLtMatmulMaybeF8(&existing_gemm).WithOneUser(),
```

Comment:
I think the logic in `CreateF8CustomCall` that supports matrix bias fusions should be removed, since it's redundant with this logic

Reply:
It's been moved to FuseMatrixBiasAdd.

Reply:
@reedwm please resolve the conversation or get back to wenscarl.